common/src/ticker.rs

```diff
@@ -0,0 +1,35 @@
+use crate::Step;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct TickerEntity {
+    pub last_ticked: Step,
+    pub ticker: Ticker,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Ticker {
```

Comment:
Due to the idea of using different components for different types of objects (even ones with a similar logic of doing something every frame), I think the right move is to be consistent in using the term "ticker" everywhere or "blinker" everywhere (probably the latter).

Reply:
Got it, `Ticker` is now `Blinker`.